cinema/serializers.py

```diff
@@ -79,15 +104,15 @@ class Meta:
             "cinema_hall_name",
             "cinema_hall_capacity",
             "tickets_available",
+            "movie_image"
         )
 
 
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], attrs["movie_session"]
-        )
+            attrs["row"], attrs["seat"], attrs["movie_session"])
```

Comment:
The `validate_ticket` method requires four arguments, but only three are provided here. Make sure to pass the `error_to_raise` argument to avoid runtime errors.